activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/extensions/ExtensionsJsonSchemaValidator.java

```diff
@@ -31,30 +33,33 @@
 /**
  * The generic JSON extensions schema for all the models
  */
-public abstract class ExtensionsJsonSchemaValidator extends JsonSchemaModelValidator implements ModelExtensionsValidator {
+public abstract class ExtensionsJsonSchemaValidator extends JsonSchemaModelValidator implements
+    ModelExtensionsValidator {
 
     @Autowired
     private JsonConverter<Model> extensionsConverter;
 
     @Override
-    public void validate(byte[] bytes,
-                         ValidationContext validationContext) {
+    public Collection<ModelValidationError> validate(byte[] bytes,
+                                                     ValidationContext validationContext) {
         super.validate(bytes,
                        validationContext);
 
         if (!validationContext.isEmpty()) {
             validateExtensionsInContext(bytes,
-                                         validationContext);
+                                        validationContext);
         }
+        return Collections.emptyList();
```

Comment:
Shouldn't this be aggregating the list of errors from `super.validate` with `validateExtensionsInContext` and returning this instead of  throwing `SemanticModelValidationException` inside `validateExtensionsInContext` and returning empty list here?